deluge/ui/gtk3/peers_tab.py

```diff
@@ -282,30 +281,17 @@ def _on_get_torrent_status(self, status):
                 if peer['progress'] != values[4]:
                     self.liststore.set_value(row, 8, peer['progress'])
             else:
-                # Peer is not in list so we need to add it
-
-                # Create an int IP address for sorting purposes
-                if peer['ip'].count(':') == 1:
-                    # This is an IPv4 address
-                    ip_int = sum(
-                        int(byte) << shift
-                        for byte, shift in zip(
-                            peer['ip'].split(':')[0].split('.'), (24, 16, 8, 0)
-                        )
-                    )
-                    peer_ip = peer['ip']
+                #requires:
+                #https://github.com/arvidn/libtorrent/commit/02884d367b5641136cf9938d27fdb8d685b32c6d
```

Comment:
this comment should be removed as it's not relevant here and should be in commit message